docs/commands/json.mset.md

```diff
@@ -37,14 +37,20 @@ For more information about replies, see [Redis serialization protocol specificat
 <summary><b>Add a new values in multiple keys</b></summary>
 
 {{< highlight bash >}}
+redis> JSON.SET doc1 $ '{"a":1}'
+OK
+redis> JSON.SET doc2 $ '{"f":{"a":2}}'
+OK
+redis> JSON.SET doc3 $ '{"f1": {"a":0}, "f2":{"a":0}}'
+OK
```

Comment:
Thank you.
Please combine the 3 `JSON.SET` commands into a single `JSON.MSET` command.